src/test/java/HW20.java

```diff
@@ -0,0 +1,19 @@
+import com.fasterxml.jackson.databind.ser.Serializers;
+import org.testng.annotations.Test;
+
+public class HW20 extends BaseTest {
+    @Test
+    public void createPlaylist2() {
```

Comment:
Adhere to the following guidelines to write expressive, easily readable test names:

- `createPlaylistTest()` is the name of the method you are testing.
- avoid "magic" number ( createPlaylistTest1, reatePlaylistTest2...)

Reply:
Ok, thank you